### PR TITLE
Feature/consistency check

### PR DESCRIPTION
**Pull request by quiteachaos**
### Release

1.0.0
https://blackminystudio.atlassian.net/browse/TPA-54
https://blackminystudio.atlassian.net/browse/TPA-52

### Description

- Created a card folder for all the card components
- Added string constants
- Removed string constants that are not used
- Edited minor errors in multiple files
- changed file names and class names according to the naming convention

### Screenshots
...

### Tips
...

---

### IMPORTANT:

- (REQUIRED) Coverage Screenshots for the tests written
- (REQUIRED) UI Screenshots for the UI implementation written

### Author preparation checklist

- [x] Current branch is rebased with latest 'main' branch
- [ ] PR is targeting only one issue
- [x] Commit messages are descriptive, consistent, and follow the project's guidelines.
- [ ] Branch name matches with at least `<jira link reference>-<short change description>`
- [x] Primary reviewer is set as assignee to the PR
- [x] No unnecessary changes on `pubspec.lock`
- [x] All added TODOs have associated Github issue references
- [x] \*.iml and, pubsepc_overrides.yaml are not checkedin

### Primary reviewer checklist

- [ ] All items in the author checklist is checked or not applicable
- [ ] All files including .lock and assets are reviewed
- [ ] Reasonable UI flows/cases related to the change are manually tested
- [ ] No commented-out code or unnecessary debug statements.
- [ ] No spelling or grammatical errors in code comments or documentation.
- [ ] No sensitive data (e.g., API keys, credentials) is hardcoded or exposed.
- [ ] The branch is up-to-date with the target branch (e.g., staging).

### Pre-merge checklist

- [ ] Two approvals received. (Changes affecting automatic PR verification, like new linter rules, formatting rules, Github workflow configurations)

---

### CODE REVIEW

**Review by krushnakantanayak — DISMISSED** [review]

File: lib/widgets/store/theme_store.dart [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+import '../../theme/theme.dart';
+import '../string_constants.dart';
+
+class ThemeStore {
+  static Color getColor(
+      {required ThemeData theme,
+      bool isReversed = false,
+      required String amount}) {
```
Comment (satyabratanayak) [inline lib/widgets/store/theme_store.dart:9]: Add comma, for better formating

File: lib/widgets/buttons/action_button.dart [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    return GestureDetector(
+      onTap: onTap,
+      child: Container(
+        padding: EdgeInsets.all(theme.spacing.width.s16),
+        decoration: BoxDecoration(
+          boxShadow:
+              title == null && icon != null ? [theme.elevation.e1] : null,
+          color: theme.colors.secondary,
+          borderRadius: title == null && icon != null
+              ? BorderRadius.circular(
```
Comment (satyabratanayak) [inline lib/widgets/buttons/action_button.dart:86]: Use comma for formatting

File: lib/widgets/cards/transaction_card.dart [hunk 3]
```diff
@@ -71,7 +72,8 @@ class TransactionCard extends StatelessWidget {
           Text(
             transactionAmount,
             style: theme.textStyle.headingSmallMedium.copyWith(
-              color: theme.colors.secondary,
+              color:
+                  ThemeStore.getColor(theme: theme, amount: transactionAmount),
```
Comment (satyabratanayak) [inline lib/widgets/cards/transaction_card.dart:76]: add comma for formatting

File: lib/widgets/store/theme_store.dart [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    final zero = amount == checkZero;
+    final minus = amount.startsWith(checkMinus);
+    if (zero) {
```
Comment (satyabratanayak) [inline lib/widgets/store/theme_store.dart:12]: add single line if statements

File: lib/widgets/buttons/action_button.dart (hunk 2 above)
Comment (quiteachaos) [reply lib/widgets/buttons/action_button.dart:86]: Done

File: lib/widgets/store/theme_store.dart (hunk 4 above)
Comment (quiteachaos) [reply lib/widgets/store/theme_store.dart:12]: done

File: lib/widgets/cards/transaction_card.dart (hunk 3 above)
Comment (quiteachaos) [reply lib/widgets/cards/transaction_card.dart:76]: done

File: lib/widgets/store/theme_store.dart (hunk 1 above)
Comment (quiteachaos) [reply lib/widgets/store/theme_store.dart:9]: done

**Review by krushnakantanayak — APPROVED** [review]

**Review by satyabratanayak — APPROVED** [review]

**Pull request merged**